pkg/auth/providers/oidc/oidc_actions.go

```diff
@@ -68,6 +72,12 @@ func (o *OpenIDCProvider) TestAndApply(request *types.APIContext) error {
 	}
 
 	oidcConfig = oidcConfigApplyInput.OIDCConfig
+	// set a default value for GroupSearchEnabled
+	// in case user input is nil for some reasons.
+	if oidcConfigApplyInput.OIDCConfig.GroupSearchEnabled == nil {
```

Comment:
shouldn't we set it to true if `oidcConfigApplyInput.OIDCConfig.GroupSearchEnabled` is true?

Reply:
This is set when the request comes with `groupSearchEnabled: true` on the line above (`oidcConfig = oidcConfigApplyInput.OIDCConfig`).
My adding is just to ensure we have a default value in case the attribute is not set for some reason.

Reply:
Ah I see, you are right. Sorry about the noise